test/integration/kubernetes.spec.ts

```diff
@@ -90,7 +101,11 @@ test('deploy sample workloads', async () => {
           argoNamespace,
         ),
       )
-      .then(() => kubectl.applyK8sYaml('./test/fixtures/argo-rollout.yaml')),
+      .then(() => kubectl.applyK8sYaml('./test/fixtures/argo-rollout.yaml'))
+      .catch((error) => {
```

Comment:
this error could come from anywhere in the chain?

Reply:
so might not be argoRollout

Reply:
No, it's only attached to `argo` flow, so irrelevant to other flows

Reply:
oh I see, this is all argo chain